README.md

```diff
@@ -1,207 +1,116 @@
 [![codecov](https://codecov.io/gh/3scale/apicast/branch/master/graph/badge.svg)](https://codecov.io/gh/3scale/apicast)
 
-APIcast is an NGINX based API gateway used to integrate your internal and external API services with 3scale’s API Management Platform.
-
-To learn more about deployment options, environments provided, and how to get started, go to the [APIcast overview](doc/overview.md).
-
-`master` branch is **not stable** and **not recommended for production** use. For the latest release, go to [Releases page](https://github.com/3scale/apicast/releases).
-
-## Description
-
-This Dockerfile creates a [3scale](http://www.3scale.net) gateway, and configures itself according to your 3scale params.
-
-## OpenShift
-
-To run APIcast on OpenShift, just use template and create a Secret to point to your 3scale Admin Portal.
-
-```shell
-oc secret new-basicauth apicast-configuration-url-secret --password=https://ACCESS-TOKEN@ACCOUNT-admin.3scale.net
-oc new-app -f https://raw.githubusercontent.com/3scale/apicast/master/openshift/apicast-template.yml
-```
-
-## Docker
+APIcast is an API gateway built on top of [NGINX](https://www.nginx.com/). It is
+part of the [Red Hat 3scale API Management
+Platform](https://www.redhat.com/en/technologies/jboss-middleware/3scale).
 
-You can download a ready to use Docker image from our repository:
+- [**Getting started**](#getting-started)
+- [**Features**](#features)
+- [**Development**](#development)
+- [**Documentation**](#documentation)
+- [**License**](#license)
 
-```shell
-docker pull quay.io/3scale/apicast:master
-```
-
-The 3scale gateway image requires one of two environment variables. The first option will pull the latest gateway configuration from the 3scale API Manager. The second points to a local configuration file which has already been downloaded from 3scale:
+## Getting started
 
-* **THREESCALE_PORTAL_ENDPOINT**
+`master` branch is **not recommended** for production use. For the latest
+release, go to the [Releases page](https://github.com/3scale/apicast/releases)
 
-URI that includes your password and portal endpoint in following format: `schema://access-token@domain`. The password can be either the [provider key](https://support.3scale.net/docs/terminology#apikey) or an [access token](https://support.3scale.net/docs/terminology#tokens) for the 3scale Account Management API. Note: these should not be confused with [service tokens](https://support.3scale.net/docs/terminology#tokens)
-Example: `https://ACCESS-TOKEN@ACCOUNT-admin.3scale.net` (where the host name is the same as the domain for the URL when you are logged into the admin portal from a browser.
+### Docker
 
-When `THREESCALE_PORTAL_ENDPOINT` environment variable is provided, the gateway will download the configuration from the 3scale on initializing. The configuration includes all the settings provided on the Integration page of the API(s).
+You need to specify your `access_token` and your `account`:
```

Comment:
`ACCESS-TOKEN` and `ACCOUNT`  to be specific?

Also, to make it work with on-premises, maybe `ACCOUNT-admin.3scale.net` > `ADMIN-PORTAL-DOMAIN`, and say that for SaaS it will be `ACCOUNT-admin.3scale.net`

I would also still mention that the Access Token should be taken from the admin portal.

Reply:
Clarified.
Thanks :+1: